linkerd/tests/common.py

```diff
@@ -338,6 +342,11 @@
     'type_overrides': LINKERD_FIXTURE_TYPES,
 }
 
+MOCK_INSTANCE_NEW = {
+    'openmetrics_endpoint': 'http://fake.tld/prometheus',
+    'extra_metrics': construct_metrics_config(LINKERD_FIXTURE_METRICS, LINKERD_FIXTURE_TYPES),
+}
```

Comment:
There's probably some work to be considered for cleaning up the new vs legacy vs v2 verbiage. Here we use "new"; the check uses "v2"; maybe somewhere else we use "legacy". Most likely for maintenance purposes we should stick to one, eg "v2".